### PR TITLE
data: WillBeebe.json

### PR DESCRIPTION
**Pull request by WillBeebe**
## Check List (Check all the applicable boxes) 

- [x] Last one until I cherry pick the Sydney out of my life. 

Edits welcome, not expected. If there's a better way to add to links or something. No worries if not. Goodnight and thank you for not telling me to stop. But you can tell me to RTFDA. <3 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5144"></a>

### CODE REVIEW

**Review by mahfuztbt — APPROVED** [review]

File: data/WillBeebe.json
Comment (priyankarpal) [inline data/WillBeebe.json:4]: looks good. but I would say that please add a small bio.

**Review by priyankarpal — CHANGES_REQUESTED** [review]
I would say that please add a small bio.

File: data/WillBeebe.json
Comment (WillBeebe) [reply data/WillBeebe.json:4]: it's too long?

Comment (kumarsonsoff3) [reply data/WillBeebe.json:4]: Yeah, it seems a lot space on your profile is covered by your bio. It would be better to keep it short & sweet! :)

image

Comment (WillBeebe) [reply data/WillBeebe.json:4]: If possible, I'd prefer to keep most of this. It's hard to get rid of my favorite words. Is Spotify the annoying part? Can I do links up there?

Comment (WillBeebe) [reply data/WillBeebe.json:4]: I'm waiting for someone to tell me to generic-link it next to the yeet coffee below. Because I personally think that's the best spot for it but I'm a very lazy person. If that's the spot for this, please don't merge this PR. I'll move them down there after I wake up. Thanks for the review!

**Comment by SurajPratap10** [comment]
> I would say that please add a small bio.

Ya, just brief in 1-2 lines

**Comment by SaraJaoude** [comment]
@WillBeebe your bio is your own and whilst there are advantages to it being concise, it should be what you want it to be.  You can always update it in the future and even use your markdown for emphasis. 

As to the Spotify links I would suggest adding this to the links section, but you could always keep it in the Bio and use markdown.

**Review by devvsakib — APPROVED** [review]
LGTM but your bio is way too long, you add your bio under 1/2 lin.

File: data/WillBeebe.json
Comment (amandamartin-dev) [reply data/WillBeebe.json:4]: Hi WillBeebe, I think your bio lenght is yours and therefore fine, but please move the link into the links section. This will be easier for folks to be able to click on it and the rendering of the text bio will look better for you too

**Review by rohitverse — CHANGES_REQUESTED** [review]
Please Add Short Bio

**Comment by SaraJaoude** [comment]
> Please Add Short Bio

@rohitverse please can you look at the conversation above before making a comment to avoid confusion and additional notifications to the author and maintainers of the project.  This point has already been discussed.  This is also the author's bio and they can have what they want. 

@WillBeebe let us know if you still want this Pull Request open whilst you decide

**Comment by rohitverse** [comment]
Okay Sorry Mam

**Comment by SaraJaoude** [comment]
Given the length of time this PR has been open I will merge it now.  If you would like to make any changes this can come in a new PR.

**Pull request merged**
